framework/include/utils/InputParameters.h

```diff
@@ -687,24 +679,51 @@ class InputParameters : public Parameters
   template <typename T, typename S>
   void setParamHelper(const std::string & name, T & l_value, const S & r_value);
 
-  /// The documentation strings for each parameter
-  std::map<std::string, std::string> _doc_string;
-
-  /// The custom type that will be printed in the YAML dump for a parameter if supplied
-  std::map<std::string, std::string> _custom_type;
-
-  /// Syntax for command-line parameters
-  std::map<std::string, std::vector<std::string>> _syntax;
-
-  /// The names of the parameters organized into groups
-  std::map<std::string, std::string> _group;
-
-  /// The map of functions used for range checked parameters
-  std::map<std::string, std::string> _range_functions;
+  struct Metadata
```

Comment:
Sorry, you will need to add leading underscores to all members here per the MOOSE coding guidelines.

Reply:
done.